build_scripts/build_usd.py

```diff
@@ -1265,6 +1604,17 @@ def InstallOpenVDB(context, force, buildArgs):
             '-DOPENVDB_BUILD_BINARIES=OFF',
             '-DOPENVDB_BUILD_UNITTESTS=OFF'
         ]
+        if context.targetIos:
+            extraArgs.append('Boost_USE_STATIC_LIBS=ON')
```

Comment:
I think the reason you are patching the boost generation code above is to support OpenVdb here. How important is the support of OpenVdb on iOS at this time? To expedite things and avoid having to support boost moving forward, would it be a reasonable approach to skip OpenVdb for now? I personally would rather not have boost on iOS more than I personal would want OpenVdb support on iOS.

Reply:
For the time being, we can skip supporting openvdb

Reply:
That's a good suggestion. By matrix, I think you are meaning documentation, as opposed to a matrix of features in the code? Perhaps the best place for it is in the documentation here https://github.com/PixarAnimationStudios/USD/blob/dev/BUILDING.md#optional-components
- that would be a separate changelist and we could update it once these changes land.

We could also emit diagnostics, as part of this changelist, so that if someone specifies for example OCIO 1.x, it could explicitly inform them of the impossibility of it.